### ISSUE TITLE
Empty string note tracks in Max fail export with a plConvert error

### ISSUE THREAD

**Issue by dpogue** [H-uru/Plasma#933]
Yes these are technically invalid, but it would be much better to just ignore note tracks that are empty instead of showing a plConvert "Unknown error during convert" popup.

**Comment by Hoikas** [comment]
Is there an already existing max file that exhibits this issue?

**Comment by dpogue** [comment]
No, I ran into this while modifying one of the Kahlo critter files for export and double-clicking in the dope view accidentally created two notes.

I can make a broken version of one of the files for testing if needed.

**Comment by dpogue** [comment]
[Example 3DS Max 7 file](https://drive.google.com/file/d/1Pmd5PPhVKOdTlUyNm83_9RJBCDSJtXo2/view)
